src/other/Table.js

```diff
@@ -55,10 +56,42 @@
         th
             .enter()
             .append("th")
-                .text(function (column) {
-                    return column;
+                .each(function(d) {
+                    var element = d3.select(this);
+                    element
+                        .append("span")
+                            .attr("class", "thText")
+                    ;
+                    element
+                        .append("span")
+                            .attr("class", "thIcon")
+                    ;
                 })
+            .on("click", function (column) {
+                if (context._currentSort === column) {
+                    if(context._order !== -1) {
+                        context._order = 1;
+                    }
+                } else {
+                    context._order = -1;
+                }
+                context.headerClick(column);
+            })
+        ;
+        th.select(".thText")
+            .text(function (column) {
+                return column;
+            })
         ;
+        th.select(".thIcon")
+            .text(function (column) {
+                if (context._order === -1) {
+                    return context._currentSort === column ? "\t" + "\uf078" : "";
```

Comment:
I don't think HTML support tab spaces "\t" - if you want a "gap" you should either style the span or add a "&nbsp;"